pkg/components/azure-arc/image/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mcr.microsoft.com/azure-cli
+
+ARG VERSION
+
+# ENV BASE_URL="https://storage.googleapis.com/kubernetes-helm"
+ENV BASE_URL="https://get.helm.sh"
+ENV TAR_FILE="helm-v${VERSION}-linux-amd64.tar.gz"
+
+
+RUN apk add --update --no-cache curl ca-certificates && \
+    curl -L ${BASE_URL}/${TAR_FILE} |tar xvz && \
+    mv linux-amd64/helm /usr/bin/helm && \
+    chmod +x /usr/bin/helm && \
+    rm -rf linux-amd64 && \
+    apk del curl && \
+    rm -f /var/cache/apk/*
+
+RUN az extension add --name connectedk8s
+
+WORKDIR /azure-arc
```

Comment:
Maybe `/usr/local/bin` ?

Reply:
I always believe `/usr/local/bin` to be a place for binaries.

In our case its a script that we intend to run , I'd rather keep it the same

Reply:
```
$ file /usr/bin/* | grep script | wc -l

1067
```